proof/refine/ARM/Schedule_R.thy

```diff
@@ -487,6 +487,22 @@ lemma ct_idle_or_in_cur_domain'_lift2:
   apply (wp static_imp_wp hoare_vcg_disj_lift | assumption)+
   done
 
+lemma valid_objs_valid_tcbs[elim!]:
```

Comment:
Please move these to TcbAcc_R, just after the definitions of `valid_tcbs` and `valid_tcbs’`.